lib/cucumber/runtime.rb

```diff
@@ -30,8 +30,12 @@ def initialize(original_exception, path)
   class FileNotFoundException < FileException
   end
 
-  class FeatureFolderNotFoundException < FileException
-    include FixRuby21Bug9285 if Cucumber::RUBY_2_1 || Cucumber::RUBY_2_2 || Cucumber::RUBY_2_3
+  class FeatureFolderNotFoundException < Exception
+    attr :path
+    def initialize(path)
+      super('No such file or directory - features')
+      @path = path
```

Comment:
Instead of calling `super`, I would suggest to override `message` method and output the path related to the error. E.g. `"Feature folder not found at #{@path}"` or similar. If we follow this path I would also use `File.expand_path(path)` for clarity.